sql/moz-fx-data-shared-prod/snowflake_migration/prospects_v1/query.sql

```diff
@@ -0,0 +1,81 @@
+WITH stg_prospects AS (
+  SELECT
+    event_id,
+    unstruct_event_com_pocket_object_update_1.trigger AS object_update_trigger,
+  -- prospect info
+    contexts_com_pocket_prospect_1[0].prospect_id AS prospect_id,
+    contexts_com_pocket_prospect_1[0].url AS url,
+    contexts_com_pocket_prospect_1[0].scheduled_surface_id AS scheduled_surface_id,
+    contexts_com_pocket_prospect_1[0].prospect_source AS prospect_source,
+    TIMESTAMP_SECONDS(contexts_com_pocket_prospect_1[0].created_at) AS created_at,
+    TIMESTAMP_SECONDS(DIV(contexts_com_pocket_prospect_1[0].reviewed_at, 1000)) AS reviewed_at,
+    contexts_com_pocket_prospect_1[0].prospect_review_status AS prospect_review_status,
+    ARRAY_TO_STRING(contexts_com_pocket_prospect_1[0].status_reasons, ",") AS status_reasons,
+    contexts_com_pocket_prospect_1[0].status_reason_comment AS status_reason_comment,
+    contexts_com_pocket_prospect_1[0].reviewed_by AS reviewed_by,
+    contexts_com_pocket_prospect_1[0].title AS title,
+    contexts_com_pocket_prospect_1[0].excerpt AS excerpt,
+    contexts_com_pocket_prospect_1[0].image_url AS image_url,
+    contexts_com_pocket_prospect_1[0].language AS LANGUAGE,
```

Comment:
```suggestion
    contexts_com_pocket_prospect_1[0].language AS language,
```

just for consistency